yt_dlp/extractor/nbc.py

```diff
@@ -584,3 +590,170 @@ def _real_extract(self, url):
             'formats': formats,
             'is_live': is_live,
         }
+
+
+class NBCStationsIE(InfoExtractor):
+    _VALID_URL = r'''(?x)
+                         https?://(?:www\.)?(?P<site>
+                             nbcbayarea|nbcboston|nbcchicago|nbcconnecticut|nbcdfw|nbclosangeles|
+                             nbcmiami|nbcnewyork|nbcphiladelphia|nbcsandiego|nbcwashington|necn|
+                             telemundo52|telemundoarizona|telemundochicago|telemundonuevainglaterra
+                         )\.com/(?:[^/]+/)*(?P<id>[^/?]+)/?(?:$|[#?])
+                    '''
+
+    _TESTS = [{
+        'url': 'https://www.nbclosangeles.com/news/local/large-structure-fire-in-downtown-la-prompts-smoke-odor-advisory/2968618/',
+        'md5': '462041d91bd762ef5a38b7d85d6dc18f',
+        'info_dict': {
+            'id': '2968618',
+            'ext': 'mp4',
+            'title': 'Large Structure Fire in Downtown LA Prompts Smoke Odor Advisory',
+            'description': None,
+            'timestamp': 1661135892,
+            'upload_date': '20220821',
+            'uploader': 'NBC 4',
+            'uploader_id': 'KNBC',
+            'channel': 'nbclosangeles',
+        },
+    }, {
+        'url': 'https://www.telemundoarizona.com/responde/huracan-complica-reembolso-para-televidente-de-tucson/2247002/',
+        'md5': '0917dcf7885be1023a9220630d415f67',
+        'info_dict': {
+            'id': '2247002',
+            'ext': 'mp4',
+            'title': 'Huracán complica que televidente de Tucson reciba reembolso',
+            'description': 'md5:af298dc73aab74d4fca6abfb12acb6cf',
+            'timestamp': 1660886507,
+            'upload_date': '20220819',
+            'uploader': 'Telemundo Arizona',
+            'uploader_id': 'KTAZ',
+            'channel': 'telemundoarizona',
+        },
+    }]
+
+    _RESOLUTIONS = {
+        '1080': '1920',
+        '720': '1280',
+        '540': '960',
+        '360': '640',
+        '234': '416',
+    }
+
+    def _real_extract(self, url):
+        channel, video_id = self._match_valid_url(url).group('site', 'id')
+        webpage = self._download_webpage(url, video_id)
+
+        nbc_data = self._search_json(
+            r'<script>var\s*nbc\s*=\s*', webpage, 'NBC JSON data', video_id)
+        pdk_acct = nbc_data.get('pdkAcct') or 'Yh1nAC'
+        fw_ssid = traverse_obj(nbc_data, ('video', 'fwSSID'))
+        fw_network_id = traverse_obj(nbc_data, ('video', 'fwNetworkID'), default='382114')
+
+        video_data = self._parse_json(self._html_search_regex(
+            r'data-videos="([^"]*)"', webpage, 'video data', fatal=False, default='{}'), video_id)
+        video_data = variadic(video_data)[0]
+        video_data.update(self._parse_json(self._html_search_regex(
+            r'data-meta="([^"]*)"', webpage, 'metadata', fatal=False, default='{}'), video_id))
```

Comment:
```suggestion
            r'data-videos="([^"]*)"', webpage, 'video data', fdefault='{}'), video_id)
        video_data = variadic(video_data)[0]
        video_data.update(self._parse_json(self._html_search_regex(
            r'data-meta="([^"]*)"', webpage, 'metadata', default='{}'), video_id))
```
default implies non-fatal